src/modules/dex/directives/tradeGraph/TradeGraph.js

```diff
@@ -103,18 +98,51 @@
                 this._poll = createPoll(this, this._getOrderBook, this._setOrderBook, 1000);
             }
 
+            /**
+             * @returns {boolean}
+             */
+            shouldShowGraph() {
+                return this._canShowGraph;
+            }
+
+            /**
+             * @returns {boolean}
+             */
+            shouldShowStub() {
```

Comment:
Those two function seem redundant while we can simply make `this._canShowGraph` property public and refer to it in the templates.